org.eclim.core/java/org/eclim/plugin/core/command/complete/AbstractCodeCompleteCommand.java

```diff
@@ -66,7 +66,7 @@ public Object execute(final CommandLine commandLine)
     if(COMPACT.equals(layout) && results.size() > 0){
       results = compact(results);
     }
-    Collections.sort(results);
```

Comment:
Did you intend to affect completion for all eclim supported languages here or were you hoping to just affect the java completions? This breaks quite a few unit tests.